Mage/src/main/java/mage/filter/predicate/card/ManaValueLessThanOrEqualToSourceQuestCounterPredicate.java

```diff
@@ -0,0 +1,26 @@
+package mage.filter.predicate.card;
+
+import mage.cards.Card;
+import mage.counters.CounterType;
+import mage.filter.predicate.ObjectSourcePlayer;
+import mage.filter.predicate.ObjectSourcePlayerPredicate;
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+
+/**
+ * @author xenohedron
+ */
+public enum ManaValueLessThanOrEqualToSourceQuestCounterPredicate implements ObjectSourcePlayerPredicate<Card> {
```

Comment:
This should be card-specific, not common, as `ArcadeGannonPredicate` in the same file as the card class